### PR TITLE
Add nsx operator version tag into util basic tags

### PR DESCRIPTION
**Pull request by timdengyun**

### CODE REVIEW

File: pkg/util/utils.go [hunk 1]
```diff
@@ -35,14 +35,16 @@ const (
...
+		common.TagScopeNamespace, common.TagScopeNamespaceUID,
 		common.TagScopeStaticRouteCRName, common.TagScopeStaticRouteCRUID,
 		common.TagScopeSecurityPolicyCRName, common.TagScopeSecurityPolicyCRUID,
 		common.TagScopeSubnetCRName, common.TagScopeSubnetCRUID,
 		common.TagScopeSubnetPortCRName, common.TagScopeSubnetPortCRUID,
 		common.TagScopeVPCCRName, common.TagScopeVPCCRUID,
 		common.TagScopeIPPoolCRName, common.TagScopeIPPoolCRUID,
-		common.TagScopeSubnetSetCRName, common.TagScopeSubnetSetCRUID}
+		common.TagScopeSubnetSetCRName, common.TagScopeSubnetSetCRUID,
+	}
```
Comment (TaoZou1) [inline pkg/util/utils.go:47]: I forget to add TagScopeCluster, TagScopeVersion here, could  you help to add them?

File: pkg/util/utils.go [hunk 2]
```diff
@@ -344,6 +346,10 @@ func BuildBasicTags(cluster string, obj interface{}, namespaceID types.UID) []mo
 			Scope: String(common.TagScopeCluster),
 			Tag:   String(cluster),
 		},
+		{
+			Scope: String(common.TagScopeVersion),
+			Tag:   String(common.TagValueVersion10),
```
Comment (dantingl) [inline pkg/util/utils.go:351]: where is TagValueVersion10, can we set value with major/minor/patch version like 1.0.0

Comment (timdengyun) [reply pkg/util/utils.go]: TagValueVersion10 is 1.0 defined in 
https://github.com/vmware-tanzu/nsx-operator/blob/vpc_dev/pkg/nsx/services/common/types.go#L64
yes, we can set 1 as major version, .x as minor version, the last .x as patch version.

File: pkg/util/utils.go (hunk 1 above)
Comment (timdengyun) [reply pkg/util/utils.go:48]: Done

File: pkg/nsx/services/common/types.go [hunk 3]
```diff
@@ -61,7 +61,9 @@ const (
 	TagValueGroupScope              string = "scope"
 	TagValueGroupSrc                string = "source"
 	TagValueGroupDst                string = "destination"
-	TagValueVersion10               string = "1.0"
+	TagValueMajorVersion            string = "1"
+	TagValueMinorVersion            string = ".0"
+	TagValuePatchVersion            string = ".0"
 	AnnotationVPCNetworkConfig      string = "nsx.vmware.com/vpc_network_config"
```
Comment (TaoZou1) [inline pkg/nsx/services/common/types.go:67]: how about:
TagValueMinorVersion  string = "0"
TagValuePatchVersion  string = "0"
define a var
	TagValueVersion10               = [3]string{TagValueMajorVersion, TagValueMinorVersion, TagValuePatchVersion}
when you need version string, 
 strings.Join(TagValueVersion10, ".")

Comment (timdengyun) [reply pkg/nsx/services/common/types.go:64]: fixed

**Review by TaoZou1 — APPROVED** [review]

**Pull request merged**